zenodo/modules/records/serializers/schemas/dc.py

```diff
@@ -30,6 +30,25 @@
 
 from ...models import ObjectType
 
+from html.parser import HTMLParser
+
+
+class HTMLStripper(HTMLParser):
+    def __init__(self):
+        self.reset()
+        self.strict = False
+        self.convert_charrefs= True
+        self.fed = []
+    def handle_data(self, d):
+        self.fed.append(d)
+    def get_data(self):
+        return ''.join(self.fed)
+
+def strip_tags(html):
```

Comment:
Running ./run_tests.sh gives me the error -> 
Warning, treated as error:
/home/rachit/zenodo/docs/api/applications.rst:30: WARNING: autodoc: failed to import module u'zenodo.factory'; the following exception was raised:
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/sphinx/ext/autodoc.py", line 551, in import_object
    __import__(self.modname)
  File "/home/rachit/zenodo/zenodo/factory.py", line 32, in <module>
    from invenio_base.app import create_app_factory
ImportError: No module named app